eng/publish/PublishVSCodeExtensionUsingMI.ps1

```diff
@@ -66,7 +66,7 @@ try {
                 if ($nugetPackageName -like '*.symbols.nupkg') {
                     Write-Host "Skipping publishing symbol package $nugetPackagePath"
                     # Use 'continue' to skip to the next iteration
-                    continue
+                    return
                 }
 
                 if ($nugetPackageName -match '(?<=(?<id>.+))\.(?<version>((\d+\.\d+(\.\d+)?))(?<suffix>(-.*)?))\.nupkg')
```

Comment:
let's fix the filter expression and remove the conditional check

Reply:
this is working.

Also this helps the script to be more verbose

https://dev.azure.com/dnceng/internal/_build/results?buildId=2570763&view=logs&j=8b2f3529-484e-5976-a626-d4a8d47f3428&t=18943edd-a66d-52d4-6098-50cd321688d9

Reply:
Closing this comment since it looks to be a duplicate